network-api/networkapi/wagtailcustomization/static/css/admin.css

```diff
@@ -53,8 +53,3 @@ fieldset ul li div label {
 #wagtail .wrapper input[type="checkbox"] {
   vertical-align: middle;
 }
-
-#wagtail .wrapper input[type="checkbox"]:before {
```

Comment:
@Pomax do you remember what this was for? I found the PR but couldn't recall the details. Was this to fix the user group checkbox misalignment? If so, checkboxes look fine without these rules now so we should be able to safely remove this part of SCSS.

![image](https://user-images.githubusercontent.com/2896608/110710266-6b57a580-81b2-11eb-8a31-6e727166cb1d.png)

Reply:
that's what we had them for, yes. If they're no longer needed, let's remove them \o/